### DISCUSSION TITLE
Open a file at startup

### DISCUSSION THREAD

**Discussion by pyhp2017** [gitpod-io/openvscode-server#434] [General]
Hey, i need to open a file at startup beside "Welcome message" in vscode. right now, if you open OpenVsCodeServer in your browser, you will see a welcome message, i want to replace that to open a "README file".

Thanks

**Comment by jeanp413** [comment]
@pyhp2017 you can use `"workbench.startupEditor"` setting and set it to `"readme"`

**Reply by pyhp2017** [reply]
if i want to change it by default i have to change it in "User & Remote" settings which is not under my control and there is no file to overwrite it.

**Reply by jeanp413** [reply]
If you want it to change the default value then you'll need to follow [this](https://github.com/gitpod-io/openvscode-server/discussions/349#discussioncomment-2787513) and compile openvscode-server yourself
